packages/core-components/src/components/LinkButton/LinkButton.tsx

```diff
@@ -43,9 +43,17 @@ const LinkWrapper = React.forwardRef<any, LinkProps>((props, ref) => (
  *
  * @public
  * @remarks
- *
- * Makes the Button to utilize react-router
  */
-export const Button = React.forwardRef<any, ButtonProps>((props, ref) => (
+export const LinkButton = React.forwardRef<any, ButtonProps>((props, ref) => (
   <MaterialButton ref={ref} component={LinkWrapper} {...props} />
 )) as (props: ButtonProps) => JSX.Element;
+
+/**
+ * @deprecated use LinkButton instead
+ */
+export const Button = LinkButton;
+
+/**
+ * @deprecated use LinkButtonProps instead
+ */
+export type ButtonProps = LinkButtonProps;
```

Comment:
These two need to be marked with `@public`

Reply:
Okay, got that taken care of.